### PR TITLE
only run preindex plugins for 'default' database

### PR DESCRIPTION
**Pull request by snopoke**
@dannyroberts 

buddy @biyeun

### CODE REVIEW

File: corehq/preindex/default_plugin.py [hunk 1]
```diff
@@ -9,16 +9,17 @@
 class DefaultPreindexPlugin(PreindexPlugin):
     def __init__(self, app_label):
         self.app_label = app_label
+        self.app_config = apps.get_app_config(self.app_label)
+        self.dir = os.path.join(self.app_config.path, '_design')
```
Comment (snopoke) [inline corehq/preindex/default_plugin.py]: `PredindexPlugin.__repr__` requires `self.dir`. This seemed like the best fix.

File: corehq/preindex/models.py [hunk 2]
```diff
@@ -3,8 +3,12 @@
 from corehq.preindex import get_preindex_plugin
 
 
-def catch_signal(sender, **kwargs):
+def catch_signal(sender, using=None, **kwargs):
     """Function used by syncdb signal"""
+    if using != 'default':
+        # only sync for the default DB
```
Comment (snopoke) [inline corehq/preindex/models.py]: This prevents calling the preindex plugins multiple times in the sharded DB setup

File: corehq/preindex/default_plugin.py (hunk 1 above)
Comment (dannyroberts) [inline corehq/preindex/default_plugin.py]: :+1:

**Comment by dannyroberts** [comment]
Simple enough!

**Pull request merged**